src/plugins/stepsform/stepsform.js

```diff
@@ -227,3 +227,80 @@ $document.on( "timerpoke.wb " + initEvent, selector, init );
 wb.add( selector );
 
 } )( jQuery, window, document, wb );
+
+( function( $, document ) {
+"use strict";
+
+// Fetch page language and set variables accordingly
+let relpreposition = " of ",
+	progressLabel = "Questionnaire progress:";
+
+// Define French progress label
+if ( wb.lang === "fr" ) {
+	relpreposition = " de ";
+	progressLabel = "Progression du questionnaire : ";
+}
```

Comment:
Please use the standard way of adding i18n text. It's already started at line 42.

You'll also need to add the text in the CSV: https://docs.google.com/spreadsheets/d/1BmMrKN6Rtx-dwgPNEZD6AIAQdI4nNlyVVVCml0U594o/edit?gid=1#gid=1

Once you have added the lines in the CSV, you'll have to run the command `grunt i18n_csv` to update the i18n.json files.